hoomd/md/methods/methods.py

```diff
@@ -187,35 +175,35 @@ class NPT(Method):
 
         box_dof(`list` [ `bool` ]): Box degrees of freedom with six boolean
             elements corresponding to x, y, z, xy, xz, yz, each. Default to
-            [True,True,True,False,False,False]). If turned on to True,
-            rescale corresponding lengths or tilt factors and components of
-            particle coordinates and velocities.
+            [True,True,True,False,False,False]). If True, rescale corresponding
+            lengths or tilt factors and components of particle coordinates and
+            velocities.
 
         rescale_all (bool): if True, rescale all particles, not only those
             in the group, Default to False.
 
         gamma (float): Dimensionless damping factor for the box degrees of
```

Comment:
Update the documentation to match the current implementation of gamma. It is has dimensions of [time^{-1}]. The below documentation should also suggest how to choose an appropriate value of gamma.